pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -137,6 +146,65 @@
+  ai_curriculum_duration_02: "Duration: 1 hour"
+  ai_curriculum_btn_aria_label_02: "View the lesson plan for AI and Ethics"
+  ai_curriculum_title_03: "AI for Oceans"
+  ai_curriculum_desc_03: "Help A.I. clean the oceans by training it to detect trash! Learn about training data and bias, and how AI can address world problems."
+  ai_curriculum_grades_03: "Grade Levels: 3-12"
+  ai_curriculum_duration_03: "Duration: 1 hour"
+  ai_curriculum_btn_aria_label_03: "Try the AI for Oceans activity"
+  ai_carousel_heading: "What teachers say about AI with Code.org:"
+  ai_carousel_quote_01: "This lesson also gave my students the opportunity to see that there is an element of opinion in classification systems. Later, when we started to see how bias can end up in AI, I was able to pull them back and remind them that the data used to train their models was based on choices."
+  ai_carousel_quote_02: "My students learned that they don't have to just be consumers when it comes to technological advances that they are experiencing on almost a daily basis. They can actually be creators!"
+  ai_pl_heading: "Preparing to teach AI"
+  ai_pl_subheading: "Prepare to teach AI using our self-paced online module."
+  ai_pl_list_01: "Get an introduction to artificial intelligence and machine learning."
+  ai_pl_list_02: "Train a machine learning model using AI Lab."
+  ai_pl_list_03: "Engage with machine learning apps."
+  ai_pl_list_04: "Consider real-world implications of AI and machine learning."
+  ai_pl_list_05: "Explore end of chapter projects."
+  ai_pl_duration: "Duration: 2 hours"
+  ai_video_series_title: "How AI Works"
+  ai_video_series_desc: "With an introduction by Microsoft CEO Satya Nadella, this series of short videos will introduce you to how artificial intelligence works and why it matters. Then delve into issues like algorithmic bias and the ethics of AI decision-making."
+  ai_resources_heading: "Additional resources"
+  ai_resources_desc: "Interested in learning more about AI? Here are some additional resources that can help extend your students' learning."
+  ai_resources_title_01: "Ethics of AI Panel"
+  ai_resources_desc_01: "Go deeper with some of our favorite AI experts! This panel discussion touches on important issues like algorithmic bias and the future of work."
+  ai_resources_btn_aria_label_01: "Watch the Ethics of AI panel"
+  ai_resources_title_02: "Imagine Cup Junior"
+  ai_resources_desc_02: "This global AI for Good challenge introduces students to Microsoft's AI for Good initiatives, empowering them to solve a problem in the world with the power of AI."
+  ai_resources_btn_aria_label_02: "Learn about the Imagine Cup Junior challenge"
+  ai_resources_title_03: "AI for Teachers"
+  ai_resources_desc_03: "Supports integration of Artificial Intelligence knowledge throughout K-12 Offer free professional development, webinars, conference presentations, and peer-to-peer learning opportunities."
```

Comment:
Is there supposed to be a period in here? ` ... throughout K-12. Offer free professional development ... `?

Reply:
Just fixed!